doc/users/customizing.rst

```diff
@@ -4,6 +4,43 @@
 Customizing matplotlib
 **********************
 
+Using style sheets
+==================
+
+Style sheets provide a means to temporarily modify configuration, but in a 
```

Comment:
> It is only temporary with in the context managers.

Or until you restart matplotlib; in IPython, they are only specific to that notebook. Point taken, though, I'll try and reword.